### PR TITLE
fix count when specifying option.id

### PR DESCRIPTION
**Pull request by nilsboy**

### CODE REVIEW

**Comment by marshallswain** [comment]
This is failing the io.js test when doing npm install on the sqlite3 module, so it's not due to this commit.  The tests pass with the two other node versions.

File: src/index.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ class Service {
     };
     
     if(count) {
-      let countQuery = this.db().count('id as total');
+      let countQuery = this.db().count(this.id + ' as total');
```
Comment (ekryski) [inline src/index.js]: we should use the es6 syntax. ``${this.id} as total``

**Comment by ekryski** [comment]
I tried kicking it again but ya this is a totally fine change. Thanks @nilsboy! A bit larger discussion but realistically we can probably remove the iojs directive. We should really be testing against v4.x and v5.x.

**Comment by marshallswain** [comment]
:shipit:

**Comment by daffl** [comment]
I removed the failing iojs in #33. I'll merge this one, fix the syntax and make a patch release. Many thanks for the PR!

**Comment by daffl** [comment]
Done. Release as v2.1.2

**Comment by nilsboy** [comment]
Thanx!

**Pull request merged**